### PR TITLE
Add the ability to delete the talent's account

### PR DESCRIPTION
**Pull request by PanfiDen**
- Wrote logic in service which deletes talent, depending on given parameter from controller.
 - Write DeleteMapping() which gets request body/request param with our talent.

### CODE REVIEW

File: src/main/java/com/softserve/skillscope/talent/TalentServiceImpl.java [hunk 1]
```diff
@@ -48,4 +50,11 @@ public Map<String, Object> getAllTalentsByPage(int page) {
             throw new BadRequestException(e.getMessage());
         }
     }
+
+    public DeletedTalent delete(Long talentId){
```
Comment (SEM24) [inline src/main/java/com/softserve/skillscope/talent/TalentServiceImpl.java:54]: Check if current talent deletes own account, not others.

File: src/main/java/com/softserve/skillscope/talent/TalentServiceImpl.java [hunk 2]
```diff
@@ -48,4 +50,11 @@ public Map<String, Object> getAllTalentsByPage(int page) {
...
+        var talent = talentRepo.findById(talentId)
```
Comment (SEM24) [inline src/main/java/com/softserve/skillscope/talent/TalentServiceImpl.java:55]: Use Talent insted var. Bad practice

**Review by SEM24 — CHANGES_REQUESTED** [review]
Check if current talent deletes own account, not others. Else is fine

File: src/main/java/com/softserve/skillscope/talent/TalentServiceImpl.java (hunk 2 above)
Comment (IlliaKyselov1902) [reply src/main/java/com/softserve/skillscope/talent/TalentServiceImpl.java:55]: I am agree with Samer

File: src/main/resources/application.properties
Comment (IlliaKyselov1902) [inline src/main/resources/application.properties:1]: pro -> prod

**Review by IlliaKyselov1902 — APPROVED** [review]

**Review by SEM24 — APPROVED** [review]

**Pull request merged**